src/core/images.js

```diff
@@ -0,0 +1,21 @@
+// Module core/images
+// This checks for images missing width and height attributes
+// and adds them if they are missing
+
+import { pub } from "core/pubsubhub";
+
+export const name = "core/images";
+
+export function run(conf, doc, cb) {
+  [...doc.querySelectorAll("img")]
+    .filter(img => !img.hasAttribute("width") || !img.hasAttribute("height"))
+    .forEach((img, i) => {
+      img.height = img.naturalHeight;
+      img.width = img.naturalWidth;
+      if (!img.id) {
+        img.id = `${i+1}-image-with-no-width-height`;
+      }
+      pub("warn", `[img](#${img.id}) is missing natural width and height`);
```

Comment:
Let's not bother warning. Let's just set it and forget it :)

Reply:
cool